### ISSUE TITLE
do not reset the camera on update

### ISSUE THREAD

**Issue by roipoussiere** [roipoussiere/cadquery-server#37]

**Comment by bernhard-42** [comment]
You could try something like

```javascript
function update_size_options() {
	options.height = window.innerHeight - 44;
	options.treeWidth = window.innerWidth > 400 ? window.innerWidth / 3 : 200;
	options.cadWidth = window.innerWidth - (options.glass ? 8 : options.treeWidth + 10);
	if (viewer != null) {
		options.position = viewer.getCameraPosition();
		options.quaternion = viewer.getCameraQuaternion();
		options.target = viewer.getCameraTarget();
		options.zoom = viewer.getCameraZoom();
	}
}
```

**Comment by roipoussiere** [comment]
Works like a charm, thank you!

fixed in 937d4ff

**Comment by bullestock** [comment]
This does not appear to work in 0.4.1 - when I modify the file in an external editor, cadquery-server reloads the file and resets the view to default.
